docs/spec/v1alpha1/imagepolicies.md

```diff
@@ -124,5 +124,73 @@ The `LatestImage` field contains the image selected by the policy rule, when it
 There is one condition that may be present: the GitOps toolkit-standard `ReadyCondition`. This will
 be marked as true when the policy rule has selected an image.
 
-[image-automation-controller]: https://github.com/image-automation-controller
```

Comment:
Are these not supported by the markdown engine?

Reply:
The same mechanism is used in the other file, so I would guess yes it is supported ..

Reply:
I can add them back, we don't use links like this in the other docs, no idea if they work with mkdocs or not.

Reply:
It's standard markdown, so I would expect it does work. I use them for keeping paragraph text clear of distracting full-line URLs.

Reply:
I added them back.